src/ChipInput.js

```diff
@@ -411,15 +463,58 @@ class ChipInput extends React.Component {
       required,
       rootRef,
       value,
+      variant,
       ...other
     } = this.props
 
     const chips = this.props.value || this.state.chips
 
-    const hasInput = (this.props.value || this.state.chips).length > 0 || this.state.inputValue.length > 0
-    const shrinkFloatingLabel = InputLabelProps.shrink != null
-      ? InputLabelProps.shrink
-      : (label != null && (hasInput || this.state.isFocused))
+    const hasInput =
+      (this.props.value || this.state.chips).length > 0 ||
+      this.state.inputValue.length > 0
+    const shrinkFloatingLabel =
+      InputLabelProps.shrink != null
+        ? InputLabelProps.shrink
+        : label != null && (hasInput || this.state.isFocused)
+
+    const InputMore = {}
```

Comment:
This could be moved to line 499 so that it is over the place where its properties are set.